### DISCUSSION TITLE
Are Base64 images supported?

### DISCUSSION THREAD

**Discussion by CBusta182** [ueberdosis/tiptap#3391] [General Questions]
Hi all. 

Like the title says, I want to know if base64 images are still supported in tiptap v2. According to the tiptap website these images are supported but after some testing and according to this [issue](https://github.com/ueberdosis/tiptap/issues/1981), it doesn't seem like it is. I just want some clarification so I know if I should implement my own support for this.

**Comment by supalarry** [comment]
Bump, I have the same question. Even though I enable base64 when configuring Image extension the image is not displayed in the Editor when I copy image form my computer and try to paste it in editor.

**Comment by justanothersynth** [comment]
I haven't tested it yet myself but it looks like @EmilBayt managed to get this working through a [custom plugin](https://github.com/ueberdosis/tiptap/issues/2912#issuecomment-1169631614).

**Comment by SawkaDev** [comment]
EDIT: Isn't this resolved by the File Handler extension? You can drop or paste images as base64?

**Comment by bdbch** [comment]
base64 should be supported, see https://tiptap.dev/docs/editor/extensions/nodes/image#allowbase64
